composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "gloudemans/shoppingcart",
+    "name": "atcliff/shoppingcart",
```

Comment:
Please remove this change. The unique identifier of THIS package is `gloudemans/shoppingcart`.